bash/moving_headers-task.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# where is our script contains
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/settingEditor.sh"
+
+# Copy all * .h files to `my-headers` directory.
```

Comment:
your script start up needs  to look like this:
./moving_headers.sh folder_name

Reply:
not correct input, use -h argument 
./moving_headers.sh  -h
Usage:
-h help
dir_name moving .h from this folder to folder ..

Reply:
./moving_headers.sh
./moving_headers.sh -h
./moving_headers.sh -e cpp 
./moving_headers.sh -d dir
./moving_headers.sh -d dir -e cpp